src/js/components/Select/SelectContainer.js

```diff
@@ -177,7 +177,7 @@ const SelectContainer = forwardRef(
                 return valueValue === optionVal;
               });
             }
-          } else if (valueKey && typeof value === 'object') {
+          } else if (valueKey && value !== null && typeof value === 'object') {
```

Comment:
Looks good. Kinda surprises me that `typeof undefined is 'undefined'` (literally a string `'undefined'`) but `typeof null` is `'object'`.

Reply:
Yep. Javascript has improve a lot in the last ten years. But, there are still quirks like this lying around.